test/diagnostics/educational-notes.swift

```diff
@@ -0,0 +1,7 @@
+// RUN: not %target-swift-frontend -enable-descriptive-diagnostics -diagnostic-documentation-path %S/test-docs/ -typecheck %s 2>&1 | %FileCheck %s
+
+extension (Int, Int) {}
+// CHECK: error: non-nominal type '(Int, Int)' cannot be extended
+// CHECK: Nominal Types
+// CHECK-NEXT: -------------
+// CHECK-NEXT: Nominal types documentation content
```

Comment:
There should probably be a test of a diagnostic with no educational notes.

Reply:
(Long-term, we should also think about whether we can `-verify` educational notes somehow, but I don't think you need that in the first PR.)